test/externalTests/common.sh

```diff
@@ -213,12 +213,13 @@ function run_test
         declare -a optimizer_settings=("{ enabled: true, details: { yul: true } }")
     fi
 
+    # Force ABIEncoderV2 for all optimization levels once.
+    force_abi_v2
```

Comment:
Do we really want to do that?

Reply:
Thinking about it again: maybe not. Ideally we would run all optimization settings with and without ABIEncoderV2, but that will increase test duration. I'd be up for doing that and removing the external tests from all tags and only run them nightly on develop. What do you think?

Reply:
I think the chances of a test failure with enabled encoder v2 and disabled optimizer that does not also happen with both enabled is very very small.

Reply:
Gnosis without any optimization fails with `InternalCompilerError: Stack too deep when compiling inline assembly: Variable headStart is 1 slot(s) too deep inside the stack.`. So we should not do it when optimizations are turned off.

Reply:
I was proposing to keep it as it was: Force abi v2 only in the last stage that contains all optimizations.

Reply:
I'm fine with that as well. Shall we close this and the initial issue then?

Reply:
Ah, wait. We still need the fix for the typo: https://github.com/ethereum/solidity/pull/6620/files#diff-7661eea32694e4b30a2266bccbc36a3eR163